aggregate/housing_production/hpd_housing_ny_affordable_housing.py

```diff
@@ -177,16 +178,10 @@ def affordable_housing_internal_review():
 
     results_citywide = citywide_hny_units_con_type(df)
 
-    print(f"finished citywide")
-
     results_borough = borough_hny_units_con_type(df)
 
-    print(f"finished borough")
-
     results_puma = PUMA_hny_units_con_type(df)
 
-    print(f"finished PUMA")
-
     # output to csv for data checks  - these need to be dumped in DO edm-recipes
 
     results_citywide.to_csv(
```

Comment:
No logic was changed here, just removing unnecessary print statement and adding double quotes to follow bandit lint style